### PR TITLE
Add integration test for Vite CSS module support

### PR DESCRIPTION
**Pull request by philipp-spiess**
Wanted to make sure stuff works with CSS modules for both the postcss and the lightningcss pipeline.

### CODE REVIEW

File: integrations/vite/css-modules.test.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { describe, expect } from 'vitest'
+import { css, html, test, ts, txt } from '../utils'
+;['postcss', 'lightningcss'].forEach((transformer) => {
```
Comment (RobinMalfait) [inline integrations/vite/css-modules.test.ts:3]: I know we do this in other places, but I think a normal for loop is a bit cleaner here (gets rid of the `;` and allows a new line again)

```suggestion
for (let transformer of ['postcss', 'lightningcss']) {
```

Or, if it gives us any benefits, we could use `describe.each` as well: https://vitest.dev/api/#describe-each

File: integrations/vite/css-modules.test.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      },
+      async ({ exec, fs }) => {
+        await exec(`pnpm vite build`)
+
+        let files = await fs.glob('dist/**/*.css')
+        expect(files).toHaveLength(1)
+        let [filename] = files[0]
+
+        await fs.expectFileToContain(filename, [
+          /\.[^f]*_foo[^t]*text-decoration-line: underline;/gi,
```
Comment (RobinMalfait) [inline integrations/vite/css-modules.test.ts:63]: Goodness gracious 😂

**Review by RobinMalfait — APPROVED** [review]
Looks good to me, just a suggestion

File: integrations/vite/css-modules.test.ts (hunk 2 above)
Comment (thecrypticace) [reply integrations/vite/css-modules.test.ts:64]: 😂😂😂

**Pull request merged**